doc/nrf/ug_thingy53.rst

```diff
@@ -260,60 +187,158 @@ You can program the precompiled firmware image using one of the following ways:
+The Nordic Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with the Nordic Thingy:53 by default include the MCUboot bootloader with serial recovery support.
+You can update applications compatible with the Nordic Thingy:53 using the DFU functionality with either `nRF Connect Programmer`_ or the `nRF Programmer`_ mobile application.
 
 Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
-For a sample or application to be compatible with Thingy:53, the application must comply with the following configuration:
+For a sample or application to be compatible with Thingy:53, the application must comply with the configurations described in the following sections.
+
+.. note::
+   To use your application with the Nordic Thingy:53 preprogrammed bootloader, you must only set the proper configuration for the Partition Manager and MCUboot.
+   Configurations that enable the other features are not mandatory, but you can enable them if needed.
+
+   All applications compatible with the Nordic Thingy:53 in the |NCS| also support the additional features.
+   Refer to the source code of the samples and applications for examples of implementation.
+
+.. _thingy53_app_partition_manager_config:
 
 Partition manager configuration
-  Thingy:53 compatible samples use :ref:`partition_manager` to define memory partitions.
-  To ensure that partition layout does not change between builds, the sample must use static partition layout that is the same as the one in Thingy:53-compatible samples in the |NCS|.
-  The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+===============================
 
-  The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
-  Trying to access data on this partition results in an ARM fault.
+The samples compatible with the Nordic Thingy:53 use :ref:`partition_manager` to define memory partitions.
+To ensure that partition layout does not change between builds, the sample must use a static partition layout that is the same for all compatible samples in the |NCS|.
+See :file:`pm_static_thingy53_nrf5340_cpuapp.yml` in a compatible sample for the configuration used.
+The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+
+The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
+Trying to access data on this partition results in an ARM fault.
+
+It is also necessary to add a flash controller overlay for the application core to allow data to be loaded from the external flash memory to the network core using the application core RAM.
```

Comment:
should we mention that this is requirement for the MCUboot ?
The image swap operation is performed by the MCUboot bootloader.